test/init_test.go

```diff
@@ -171,5 +171,14 @@ func getCRDYaml(cs *clients) ([]byte, error) {
 	for _, i := range trs.Items {
 		printOrAdd("TaskRun", i.Name, i)
 	}
+
+	pods, err := cs.KubeClient.Kube.CoreV1().Pods(ns).List(metav1.ListOptions{})
+	if err != nil {
+		return nil, err
```

Comment:
could the error message here be augmented with some context, e.g. something like `couldn't get pods from namespace %s`

(that doesnt seem to be the norm in this function but i generally find that more context helps whenever posisble!)

Reply:
I will update that function to have helper msg